experimental/dds/tree2/src/feature-libraries/sequence-field/invert.ts

```diff
@@ -345,13 +351,23 @@ function applyMovedChanges<TNodeChange>(
 	if (entry.length < mark.count) {
 		const [mark1, mark2] = splitMark(mark, entry.length);
 		const mark1WithChanges =
-			entry.value !== undefined ? withNodeChange(mark1, entry.value) : mark1;
+			entry.value !== undefined
+				? withNodeChange<CellMark<MoveSource, TNodeChange>, MoveSource, TNodeChange>(
```

Comment:
I wonder if there's some way we can restrict the type parameter of `withNodeChange` so that this explicit parameterization isn't necessary. I played with this a little last month but couldn't figure out how to do it.

Reply:
It's only needed in some rare cases. I'm not even sure how it is that it is needed in this case.